### DISCUSSION TITLE
Organization of apps

### DISCUSSION THREAD

**Discussion by evok01** [Budibase/budibase#10835] [Ideas]
Is there any way to organize the list of applications into groups somehow? We are already amassing a large number of apps, but many of the apps share characteristics such as what department they are going to be used by, and what internal services they touch.

Would it be possible to group these apps into some kind of visual container on the app list, or maybe apply tags to each app? I had looked around the interface for something like this but hoping someone else knows.

**Comment by melohagan** [comment]
Hey @evok01 

This is a good idea! There is an open issue for the concept of favourite apps here: https://github.com/Budibase/budibase/issues/7472

I imagine this would fall into that work.

**Reply by melohagan** [reply]
Another trick is to name your apps with a group at the start of the name. The apps are ordered alphabetically by default.

For example:

`hr/adminpanel`
`hr/holidaybooking`
`management/adminpanel`
`management/dashboard`

etc
